From 9edd6634647c5c7076cdf90402f7603527eb7f4d Mon Sep 17 00:00:00 2001
From: Brian Lamar <brian.lamar@rackspace.com>
Date: Mon, 24 Oct 2011 16:25:21 -0500
Subject: [PATCH] Removed 'mox==0.5.0' and replaced with just 'mox' in
 tools/pip-requires.

I was getting an error from pip stating: Could not find a version that
satisfies the requirement mox==0.5.0 (from versions: ).

Potentially pip is messed up, but all tests seem to run with the latest
version of mox (0.5.3).

(cherry picked from commit d521d6529e0d0cdd1138823e0ee5be0e5d87e21c)

Change-Id: Id6a38766290a20d752177e8d272284b7b632156b
---
 tools/pip-requires |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/pip-requires b/tools/pip-requires
index f928299..e439219 100644
--- a/tools/pip-requires
+++ b/tools/pip-requires
@@ -12,7 +12,7 @@ nose
 nose-exclude
 sphinx
 argparse
-mox==0.5.0
+mox
 boto
 swift
 -f http://pymox.googlecode.com/files/mox-0.5.0.tar.gz
-- 
1.7.6.5

